packaging/kibana.spec

```diff
@@ -24,32 +24,32 @@ mkdir %{name}
 cd %{name}
 
 #extract sources
-tar xzf %_sourcedir/%{name}-%{version}.tar.gz
+tar xf %_sourcedir/%{name}-%{version}.tar
 if [ $? -ne 0 ]; then
    exit $?
 fi
 
 #extract dlumbrer/kbn_network plugin, following the renaming and file modification steps from github repo
-unzip -q resources/plugins/kbn_network*.zip -d plugins/
-if [ $? -ne 0 ]; then
-   echo "Exiting build. Could not unzip plugin."
-   exit 1
-fi
-mv plugins/kbn_network* plugins/network_vis
-rm -rf plugins/network_vis/images/
+#unzip -q resources/plugins/kbn_network*.zip -d plugins/
+#if [ $? -ne 0 ]; then
+#   echo "Exiting build. Could not unzip plugin."
+#   exit 1
+#fi
+#mv plugins/kbn_network* plugins/network_vis
+#rm -rf plugins/network_vis/images/
 
 %build
 #must install kibana dependencies before running build due to a `yarn kbn bootstrap` bug that strips auth
 cd %{name}
 /usr/bin/yarn
 
 #plugin has a package.json that must also be installed before running build
-cd plugins/network_vis/
+#cd plugins/network_vis/
 /usr/bin/yarn
 
 #run the build
-cd ../../
-/usr/bin/yarn kbn bootstrap
+#cd ../../
+/usr/bin/yarn kbn bootstrap --allow-root 
```

Comment:
This change is the only one I was surprised by. Can you explain this to me?

Reply:
When building on a build server using fab, the build is done as root. The Kibana build has added special code to check for root user during the build. This was an attempt to get the build to work. I've since reverted this, and moved the "--allow-root" to the correct location in the package.json file.